model/job.go

```diff
@@ -263,7 +326,16 @@ func (j *Job) WriteConfigs(role *Role, lightOpinionsPath, darkOpinionsPath strin
 	if err != nil {
 		return nil, err
 	}
-	config["properties"] = properties
+	config.Properties = properties
+
+	for _, provider := range j.LinkProvides {
+		config.ExportedProperties = append(config.ExportedProperties, provider.Properties...)
```

Comment:
Is it ok if there a multiple entries for the same property in this list? If not, wouldn't this be better as a map?

Reply:
I don't think it would actually _hurt_.